packages/icons/manifest.json

```diff
@@ -1968,6 +1968,11 @@
       "name": "sum_up_logomark",
       "category": "Brand",
       "size": "24"
+    },
+    {
+      "name": "forward",
+      "category": "Miscellaneous",
+      "size": "24"
     }
```

Comment:
Please follow the convention of grouping the icons by category and sorting them alphabetically.